src/hooks/useDarkMode.js

```diff
@@ -0,0 +1,12 @@
+import useLocalStorage from "./useLocalStorage"
+import { useEffect } from 'react'
+
+const useDarkMode = () => {
+    const [someValue, setSomeValue] = useLocalStorage('key')
+    useEffect(() => {
+        someValue ? document.body.classList.add("dark-mode") : document.body.classList.remove("dark-mode")
```

Comment:
if someValue = true add dark mode to the page else remove dark mode from the page